modules/graph/graph.go

```diff
@@ -0,0 +1,76 @@
+package graph
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	a "vsc-node/modules/aggregate"
+
+	"github.com/99designs/gqlgen/graphql/handler"
+	"github.com/99designs/gqlgen/graphql/playground"
+)
+
+// type Graph interface{}
+type graph struct {
+	server *handler.Server
+	port   string
+	//
+	done chan bool
+}
+
+var _ a.Plugin = &graph{}
+
+func New() *graph {
+	return &graph{
+		port: defaultPort,
+		done: make(chan bool),
+	}
+}
+
+const defaultPort = "8080"
+
+func (g *graph) graohQLResolver(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+	}
+
+	g.server.ServeHTTP(w, r)
+}
+
+func (g *graph) playgroundHandler(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodGet {
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+		return
+	}
+	playground.ApolloSandboxHandler("GraphQL playground", "/query").ServeHTTP(w, r)
+}
+
+func (g *graph) Init() error {
+	g.server = handler.NewDefaultServer(NewExecutableSchema(Config{Resolvers: &Resolver{}}))
+
+	http.HandleFunc("/", g.playgroundHandler)
+	http.HandleFunc("/query", g.graohQLResolver)
```

Comment:
Both these route definitions are wrong.

- This is where you should define GET & POST request filters. https://pkg.go.dev/net/http#Handle
- The routes should match.
- The routes should be the same as the current VSC node. https://github.com/vsc-eco/vsc-node/tree/main/src/modules/api